numpyro/mcmc.py

```diff
@@ -83,7 +83,7 @@ def init_kernel(init_samples,
             hmc_state, _ = fori_loop(0, num_warmup_steps, warmup_update, (hmc_state, wa_state))
             return hmc_state
         else:
-            return hmc_state, wa_state, warmup_update
+            return hmc_state, wa_state, wa_update
```

Comment:
I want to use wa_update instead because warmup_update uses non-jit version of sample_kernel.